rigelprotocol/SmartswapDappV2
# Remove liquidity implementation

#### What does this PR do?
- This PR implements the remove liquidity function

#### What has been completed?

#### How should the changes be manually tested?
- Go on the liquidity page, select a liquidity to remove, input the percentage you want to remove, approve tokens if you haven't and confirm withdrawal

#### Any Background context or information you want to provide?
-

#### Demo Video and screenshots?
- https://www.loom.com/share/2143561668d14301a7514d17d09b8259
